Source/Core/DolphinQt/Config/ARCodeWidget.cpp

```diff
@@ -123,6 +126,24 @@ void ARCodeWidget::SortAlphabetically()
   OnListReordered();
 }
 
+void ARCodeWidget::SortEnabledCodesFirst()
+{
+  std::sort(m_ar_codes.begin(), m_ar_codes.end(),
+            [](const auto& a, const auto& b) { return a.enabled && a.enabled != b.enabled; });
```

Comment:
Maybe `std::stable_sort` so the order within the enabled/disabled set gets preserved?